packages/coinjoin/src/backend/CoinjoinBackendClient.ts

```diff
@@ -38,6 +40,7 @@ export class CoinjoinBackendClient {
         this.blockbookUrls = arrayShuffle(settings.blockbookUrls);
         this.blockbookRequestId = Math.floor(Math.random() * settings.blockbookUrls.length);
         this.websockets = new CoinjoinWebsocketController(settings);
+        this.emitter = new TypedEmitter<{ mempoolDisconnected: void }>();
```

Comment:
woulnt be easier to store only `onDisconnect` callback? and set/unset it by mempool subscription functions?
this looks a little bit confusing and  to me

Reply:
I'd definitely be easier, but it works only for single `onDisconnect` callback and therefore only one mempool subscriber (while `subscribeMempoolTxs` in theory allows multiple listeners).

But it's true that we aren't subscribing to mempool from multiple places at once, so I could rework it a bit to better indicate that only one subscriber is expected.

Reply:
In the end, I've just added some explanation, I hope it's ok.

Reply:
looks good, thanks